test/cljam/t_bam.clj

```diff
@@ -40,23 +45,96 @@
 ;;     (is (not-throw? (spit-bam-for-test temp-file
 ;;                                        (slurp-bam-for-test large-bam-file))))))
 
+(defn- shallow= [alns1 alns2]
+  (when (= (count alns1) (count alns2))
+    (loop [alns1 alns1
+           alns2 alns2]
+      (let [aln1 (first alns1)
+            aln2 (first alns2)]
+        (if (nil? aln1)
+          true
+          (when (and (= (:rname aln1) (:rname aln2))
+                     (= (:pos aln1) (:pos aln2)))
+            (recur (rest alns1) (rest alns2))))))))
```

Comment:
Thanks for review.
I guess sure too, I fixed https://github.com/chrovis/cljam/pull/60/commits/88ff657d168b02822558b460af2eb4bb1b369326 .